go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/adrg/xdg v0.2.1
 	github.com/anchore/go-testutils v0.0.0-20200925183923-d5f45b0d3c04
 	github.com/anchore/go-version v1.2.2-0.20200810141238-330bef18dbca
-	github.com/anchore/grype-db v0.0.0-20210322113357-5aec8a7cb962
+	github.com/anchore/grype-db v0.0.0-20210525151544-6d0ab949b8fb
```

Comment:
@wagoodman I had to perform surgery here to make this work after rebasing from `main`. Could you triple check this work is right?

Reply:
looks alright 👍